supervisor/data/syslog-identifiers.json

```
@@ -0,0 +1,34 @@
+[
+  "NetworkManager",
+  "bluetoothd",
+  "bthelper",
+  "btuart",
+  "dbus-broker",
+  "dockerd",
+  "dropbear",
+  "fstrim",
+  "hassos-apparmor",
+  "hassos-config",
+  "hassos-expand",
+  "hassos-overlay",
+  "hassos-persists",
```

Can we add `hassos-supervisor` here too? It sometimes has interesting information what the systemd service did to the Supervisor container before starting:
```suggestion
  "hassos-persists",
  "hassos-supervisor",
```

---

I thought pascal said that one just replicates the supervisor logs? so youll end up seeing all supervisor logs in `/host/logs` as well

---

That used to be the case back in the days, but changed in in HAOS 7.0. Today you only get output of the [hassos-supervisor](https://github.com/home-assistant/operating-system/blob/9.2/buildroot-external/rootfs-overlay/usr/sbin/hassos-supervisor) script, which prints some useful information in certain situations.

---

👍